src/plugins/curator/screens/timeline/index.js

```diff
@@ -14,6 +14,7 @@ class TimelineScreen extends AbstractNodeScreen {
   getTabs() {
     return [
       'details',
+      delegate.getSchemas().node.hasMixin('triniti:common:mixin:custom-code') && 'code',
```

Comment:
@albertgcsula use schemas that is already available (like the seo check below)

Reply:
@gdbrown updated.